3.preprocessing_features/scripts/1.sc_quality_control.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform single-cell quality control
+# 
+# In this notebook, we perform single-cell quality control using coSMicQC. We filter the single cells by identifying outliers with z-scores, and use either combinations of features or one feature for each condition. We use features from the AreaShape and Intensity modules to assess the quality of the segmented single-cells:
+# 
+# ### Assessing poor nuclei segmentation
+# 
+# Due to high confluence with various seeding densities, sometimes nuclei overlap on top of each other, creating highly intense clusters within the Hoechst channel. To identify these nuclei, we use:
```

Comment:
Are we expecting better results from lower seeding densities?

Reply:
We can not say lower seeding density universally, as we must make this decision on a cell-line basis. Some cell lines might perform better at a higher seeding density if they don't proliferate very quickly, while others might be better at lower seeding densities cause they are less likely to overlap. 

In my QC, I tend to notice that high confluence comes from higher seeding densities, but a cell line might be highly confluent at a low seeding density but at a higher time point (e.g., after 72 hours).